polymetis/polymetis/python/polymetis/robot_client/robotiq_gripper/robotiq_gripper_client.py

```diff
@@ -6,21 +6,26 @@
 from concurrent import futures
 
 import grpc
+from google.protobuf import timestamp_pb2
 
 import polymetis_pb2
 import polymetis_pb2_grpc
 
+from polymetis.utils import Spinner
 from .third_party.robotiq_2finger_grippers.robotiq_2f_gripper import (
     Robotiq2FingerGripper,
 )
 
 
-class RobotiqGripperServer(polymetis_pb2_grpc.GripperServerServicer):
+class RobotiqGripperClient:
     """gRPC server that exposes a Robotiq gripper controls to the client
```

Comment:
This documentation is not up to date, right? Because this is no longer a gRPC server.